apiAuditLogs/apiAuditLogs.go

```diff
@@ -0,0 +1,277 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package apiAuditLogs
+
+import (
+	"fmt"
+	"go/ast"
+	"go/token"
+	"go/types"
+	"strings"
+
+	"github.com/mattermost/mattermost-govet/facts"
+	"golang.org/x/tools/go/analysis"
+	"golang.org/x/tools/go/analysis/passes/inspect"
+	"golang.org/x/tools/go/ast/inspector"
+)
+
+var Analyzer = &analysis.Analyzer{
+	Name:     "apiAuditLogs",
+	Doc:      "check the audit logs usage in the API",
+	Requires: []*analysis.Analyzer{inspect.Analyzer, facts.ApiHandlerFacts},
+	Run:      run,
+}
+
+func run(pass *analysis.Pass) (interface{}, error) {
+	whiteList := map[string]bool{
```

Comment:
I would move this to a separate file `whitelist.go` with a global variable. It's too long to remain in the same file.

Reply:
done